src/stats.js

```diff
@@ -1,45 +1,15 @@
 import * as init from './init';
-import { addCacheEventListener } from './event';
-import { HTTP_5XX_RESPONSE, actions } from './consts';
+import { HTTP_5XX_RESPONSE } from './consts';
 
-export default async function stats(emitter, upstreamName, context, next) {
+export default (emitter, upstreamName) => async (context, next) => {
```

Comment:
The `emitter` is no longer needed, but apparently the `upstreamName` still is. These parameters are no longer bound from the `index` file, which means they won't be passed through.

Reply:
Yeah I think we can remove the emitter. They are still being passed through, because this function is basically the wrapper to the plugin function, so in the plugin we'd call:

`.use(stats(emitter, upstreamName))` 

which would then call the plugin function with `context` and `next`, as you mentioned up above 👍

Reply:
https://github.com/bbc/http-transport-emf-stats/pull/9#discussion_r482951599

Yep, exactly 👍